app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,55 @@
+# user_reviews GET    /users/:user_id/reviews(.:format)                reviews#index
+#              POST   /users/:user_id/reviews(.:format)                reviews#create
+# new_user_review GET    /users/:user_id/reviews/new(.:format)            reviews#new
+# edit_user_review GET    /users/:user_id/reviews/:id/edit(.:format)       reviews#edit
+#  user_review GET    /users/:user_id/reviews/:id(.:format)            reviews#show
+#              PATCH  /users/:user_id/reviews/:id(.:format)            reviews#update
+#              PUT    /users/:user_id/reviews/:id(.:format)            reviews#update
+#              DELETE /users/:user_id/reviews/:id(.:format)            reviews#destroy
+#
+#
+#              product_reviews GET    /products/:product_id/reviews(.:format)          reviews#index
+#                 POST   /products/:product_id/reviews(.:format)          reviews#create
+# new_product_review GET    /products/:product_id/reviews/new(.:format)      reviews#new
+# edit_product_review GET    /products/:product_id/reviews/:id/edit(.:format) reviews#edit
+#  product_review GET    /products/:product_id/reviews/:id(.:format)      reviews#show
+#                 PATCH  /products/:product_id/reviews/:id(.:format)      reviews#update
+#                 PUT    /products/:product_id/reviews/:id(.:format)      reviews#update
+#                 DELETE /products/:product_id/reviews/:id(.:format)      reviews#destroy
+class ReviewsController < ApplicationController
+
+  helper_method :top_ratings, :display_name
```

Comment:
Do these helper methods work? I thought that they had to match a method that exists in the file.